apps/opt.c

```diff
@@ -326,6 +326,30 @@ int opt_int(const char *value, int *result)
     return 1;
 }
 
+static STRSTR_PAIR base[] = {
+    {"0x", "hex"},
+    {"0X", "hex"},
+    {"0", "oct"},
+    {NULL}
+};
+
+#define opt_number_error(v, b) \
```

Comment:
Oops, I didn't read the coding style file carefully and misunderstood it as macro resembled functions being preferred (it's preferable in fact...).
I will fix this.